assets/scripts/app/load_resources.js

```diff
@@ -126,23 +86,88 @@ function loadSVGs () {
 }
 
 /**
- * Wraps `new Image()`'s onload and onerror properties with
- * Promises. This provides a more reliable way of knowing when a
- * image element is done loading (rather than fetch) because a
- * fetch can successfully load an image file but be resolved
- * before the image element is ready. So we must resolve only after
- * the image's `onload` callback has been called.
+ * Gets a string representation of an <svg> via its outerHTML property
+ * or with a fallback method if outerHTML is unavailable (IE / Edge)
+ *
+ * @param {SVGElement} svg
+ * @return {string}
+ */
+function getSVGOuterHTML (svg) {
+  let outerHTML = svg.outerHTML
+
+  // If `outerHTML` is not available, use this alternate method below
```

Comment:
When would outerHTML not be available? Maybe we can add a comment about why this is necessary

Reply:
It's not available in IE/Edge, and I had the comment higher up in the JSDoc comment but I'll move it into the function.